database/redis/contact_notification_history.go

```diff
@@ -160,3 +161,30 @@ func (connector *DbConnector) CleanUpOutdatedNotificationHistory(ttl int64) erro
 		return nil
 	})
 }
+
+// CountEventsInNotificationHistory returns the amount of events in time range (from, to) for given contact ids.
```

Comment:
Amount для неисчислимых

```suggestion
// CountEventsInNotificationHistory returns the number of events in time range (from, to) for given contact ids.
```

Reply:
Не знал, спасибо